ibmcloudshellv1/ibm_cloud_shell_v1_examples_test.go

```diff
@@ -0,0 +1,190 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ibmcloudshellv1_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/ibmcloudshellv1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+//
+// This file provides an example of how to use the IBM Cloud Shell service.
+//
+// The following configuration properties are assumed to be defined:
+// IBM_CLOUD_SHELL_URL=<service base url>
+// IBM_CLOUD_SHELL_AUTH_TYPE=iam
+// IBM_CLOUD_SHELL_APIKEY=<IAM apikey>
+// IBM_CLOUD_SHELL_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+const externalConfigFile = "../ibm_cloud_shell_v1.env"
+
+var (
+	ibmCloudShellService *ibmcloudshellv1.IbmCloudShellV1
+	config               map[string]string
+	configLoaded         bool = false
+)
+
+func shouldSkipTest() {
+	if !configLoaded {
+		Skip("External configuration is not available, skipping tests...")
+	}
+}
+
+var _ = Describe(`IbmCloudShellV1 Examples Tests`, func() {
+	Describe(`External configuration`, func() {
+		It("Successfully load the configuration", func() {
+			var err error
+			_, err = os.Stat(externalConfigFile)
+			if err != nil {
+				Skip("External configuration file not found, skipping tests: " + err.Error())
+			}
+
+			os.Setenv("IBM_CREDENTIALS_FILE", externalConfigFile)
+			config, err = core.GetServiceProperties(ibmcloudshellv1.DefaultServiceName)
+			if err != nil {
+				Skip("Error loading service properties, skipping tests: " + err.Error())
+			}
+
+			configLoaded = len(config) > 0
+		})
+	})
+
+	Describe(`Client initialization`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It("Successfully construct the service client instance", func() {
+			var err error
+
+			// begin-common
+
+			ibmCloudShellServiceOptions := &ibmcloudshellv1.IbmCloudShellV1Options{}
+
+			ibmCloudShellService, err = ibmcloudshellv1.NewIbmCloudShellV1UsingExternalConfig(ibmCloudShellServiceOptions)
+
+			if err != nil {
+				panic(err)
+			}
+
+			// end-common
+
+			Expect(ibmCloudShellService).ToNot(BeNil())
+		})
+	})
+
+	Describe(`IbmCloudShellV1 request examples`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`GetAccountSettingsByID request example`, func() {
+			fmt.Println("\nGetAccountSettingsByID() result:")
+			// begin-get_account_settings_by_id
+
+			getAccountSettingsByIdOptions := ibmCloudShellService.NewGetAccountSettingsByIdOptions(
+				AccountID: core.StringPtr("12345678-abcd-1a2b-a1b2-1234567890ab"),
```

Comment:
We discussed configuring this value in the .env file.

Reply:
Done